moduleroot/.travis.yml.erb

```diff
@@ -106,11 +106,34 @@ matrix:
       <%= key %>: <%= job[key].gsub(/@@SET@@/, set['set']).gsub(/@@COLLECTION@@/, set.fetch('collection', 'puppet6')).gsub(/@@TESTMODE@@/, set.fetch('testmode', 'apply')) %>
 <%   end -%>
 <% end -%>
+<% if @configs['use_litmus']['enabled']
+  @configs['use_litmus']['puppet_collection']||=['puppet5', 'puppet6']
+  @configs['use_litmus']['provision_list']||=['travis_el','travis_deb']
+  @configs['use_litmus']['rvm']||='2.5.3'
+  @configs['use_litmus']['puppet_collection'].each do |puppet_version|
+  @configs['use_litmus']['provision_list'].each do |platform|-%>
+    -
```

Comment:
typo?  Should `-` be there?

Reply:
Hey @glennsarti ,
I tried to keep the same structure as the current one for travis config file.
![Screenshot 2019-12-11 at 10 46 15](https://user-images.githubusercontent.com/42546087/70608930-a4991280-1c09-11ea-9668-25e5022e2e60.png)

Reply:
It's not in scope for this PR, but I've always been disappointed by the formatting of YAML files in general. Output and coding would be a lot nicer and safer if the whole file got changed into a single codeblock at the start building a datastructure and then the last line would only `YAML.dump` that datastructure.